src/userRepo.js

```diff
@@ -60,28 +53,28 @@ class UserRepo {
   }
 
   findGoodSleepers(startDate) {
```

Comment:
I'm not sure what this function is doing. What's the logic here?

Reply:
We were told to find the people whose sleepQuality averaged above a 3 for a given week.